state/execution.go

```diff
@@ -256,8 +256,7 @@ func execBlockOnProxyApp(
 
 	valUpdates := abciResponses.EndBlock.ValidatorUpdates
 	if len(valUpdates) > 0 {
-		// TODO: cleanup the formatting
-		logger.Info("Updates to validators", "updates", valUpdates)
+		logger.Info("Updates to validators", "updates", cdc.MustMarshalJSON(valUpdates))
```

Comment:
So this now gives us

```
I[15116-11-15|14:36:25.897] Updates to validators                        module=state updates="[{\"pub_key\":{\"type\":\"ed25519\",\"data\":\"vGBScHpFk+2oPrE7x6hSiSI/8gPOWfovPqW2klMHsyo=\"},\"power\":\"1\"}]"
```

Probably better than before, but maybe would be even better if we just built a string that looked like `<addr1>:<power1>,<addr2>:<power2>,...`, instead of the JSON with all the extra escaped strings.

We can also put it in `updateState` instead of in here, right after updateValidators is called.

 Thoughts?